doc/hash/hash-design.md

```diff
@@ -689,6 +783,30 @@ module sonic-hash {
         description "First Revision";
     }
 
+    typedef hash-field {
+        description "Represents native hash field";
+        type stypes:hash-field {
+            enum IN_PORT;
+            enum DST_MAC;
+            enum SRC_MAC;
+            enum ETHERTYPE;
+            enum VLAN_ID;
+            enum IP_PROTOCOL;
+            enum DST_IP;
+            enum SRC_IP;
+            enum L4_DST_PORT;
+            enum L4_SRC_PORT;
+            enum INNER_DST_MAC;
+            enum INNER_SRC_MAC;
+            enum INNER_ETHERTYPE;
+            enum INNER_IP_PROTOCOL;
+            enum INNER_DST_IP;
+            enum INNER_SRC_IP;
+            enum INNER_L4_DST_PORT;
+            enum INNER_L4_SRC_PORT;
+        }
+    }
+
     container sonic-hash {
```

Comment:
Please add the revision statement in the Yang PR.

Reply:
@prvattem sure. Will do.